resotoworker/resotoworker/config.py

```diff
@@ -1,19 +1,47 @@
+from concurrent.futures import ThreadPoolExecutor, TimeoutError
+from collections import namedtuple
 from resotolib.graph import GraphMergeKind
 from resotolib.config import Config
 from resotolib.proc import num_default_threads
+from resotolib.baseplugin import BaseCollectorPlugin
+from resotolib.logger import log
 from attrs import define, field
-from typing import ClassVar, Optional, List
+from typing import ClassVar, Optional, List, Type
 
 
-def add_config(config: Config) -> None:
+_collector_plugins: List[Type[BaseCollectorPlugin]] = []
+PluginAutoEnabledResult = namedtuple("PluginAutoEnabledResult", ["cloud", "auto_enabled"])
```

Comment:
What do you think about a frozen attrs class instead? e.g.

```python
@frozen
class PluginAutoEnabledResult:
    cloud: str
    auto_enabled: bool
```

Reply:
Great idea, will do!